src/views/admin/index.hbs

```diff
@@ -0,0 +1,15 @@
+<div class="jumbotron mt-1">
+     <div class="container" style="text-align: left;">
+
+          <h1>Admin Module</h1>
+          <!--linea -->
+          <hr color="blue" size=3>
+           <h4>AT-SCE</h4>
+
+           <button type="button" class="btn btn-primary"> User</button>
```

Comment:
`<a class="btn btn-primary" href="/admin/user">&raquo; Users</a>`

Reply:
you can follow this view as reference https://github.com/at-internship/at-sso-ui/blob/develop/src/views/admin/index.hbs